src/components/text-area/text-area.tsx

```diff
@@ -317,13 +312,13 @@ export class TextArea
   @Method()
   async setFocus(): Promise<void> {
     await componentLoaded(this);
-    this.textareaEl.focus();
+    this.textAreaEl.focus();
   }
 
   /** Selects all text of the component's `value`. */
```

Comment:
+@geospatialem for editorial review.

Reply:
matched with `calcite-input` but i feel this can be modified to `Selects the text of the component ` may be.

Reply:
This and `input` does read a little difficult (I'll tackle that one in a different PR).

Concur with the proposal you have above:
> Selects the text of the component.